packages/compartment-mapper/src/zip/format-writer.js

```diff
@@ -49,7 +49,7 @@ const textEncoder = new TextEncoder();
  */
 function writeDosDateTime(writer, date) {
   const dosTime =
-    date !== undefined
+    date !== undefined && date !== null
```

Comment:
I assume lint:ts motivates this. Good, good.

Reply:
yes...

I buried this in the commit comment: `Date?` evidently includes `null` as well as `undefined`

doesn't seem like it's worth a code comment, but I could easily be persuaded otherwise

Reply:
No need for a change here.